android/app/build.gradle

```diff
@@ -94,7 +94,7 @@ unzipDeps.dependsOn makeLocalization
 preBuild.dependsOn unzipDeps
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 26
```

Comment:
These changes are unrelated.

Reply:
You need to revert changes made in gradle file. If you want them merged - they should go to separate PR.